FreeRTOS/Test/CMock/smp/multiple_priorities_no_timeslice/covg_multiple_priorities_no_timeslice_utest.c

```diff
@@ -627,4 +634,758 @@ void test_task_delay_until_with_config_assert( void )
 
     xTaskDelayUntil( &previousWakeTime , 4 );
     
-}
\ No newline at end of file
+}
+
+/**
+ * @brief xTaskGenericNotify - function to notify a task.
+ *
+ * <b>Coverage</b>
+ * @code{c}
+ *              case eNoAction:
+ *              ...
+ *                    break;
+ *
+ * @endcode
+ *
+ *  Cover the eNoAction action type with an explicit task specified.
+ */
+void test_coverage_xTaskGenericNotify_with_eAction_equalto_eNoAction( void )
+{
+    TCB_t xTaskTCBs[ configNUMBER_OF_CORES + 1U ] = { NULL };
+    UBaseType_t xidx = 0;
+    uint32_t *prevValue = NULL;
+    uint32_t ulValue = 0;
+    uint32_t i;
+    BaseType_t xReturn;
+
+    /* Setup the variables and structure. */
```

Comment:
The xTaskGenericNofiy API doesn't involve these variables. Please reference this commit for suggestion and applied to this suggestion with other test cases in this PR.
https://github.com/chinglee-iot/FreeRTOS/commit/c31a01b0e30cada481b6bf674399d331eac5f3b3